src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int FIGURE_COUNT = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[FIGURE_COUNT];
+        for (int i = 0; i < FIGURE_COUNT; i++) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
```suggestion
            System.out.println(balls[i]);
```
toString() is called automatically